App.js

```diff
@@ -1,19 +1,40 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import 'react-native-gesture-handler'
+import * as React from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import { createStackNavigator } from '@react-navigation/stack'
 
-export default function App() {
+import LoginScreen from './src/screen/LoginScreen'
+import RegisterScreen from './src/screen/RegisterScreen'
+import ListScreen from './src/screen/ListScreen'
+import { createStore } from './node_modules/redux'
+import {Provider} from 'react-redux'
+import taskAdd from './src/reducers'
+
+const Stack = createStackNavigator()
+const store = createStore(taskAdd)
+
+function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-    </View>
-  );
+   <Provider store={store}>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="LoginScreen" screenOptions={{ headerShown: false, transitionSpec: { open: config, close: config } }} >
```

Comment:
Ponto de observação. A tela de início deveria ser a tela `ListScreen` e ela decidir se vai pro login ou não.
Pq senão, vc sempre vai mostrar a tela de login, coisa que não queremos quando o usuário já estiver logado.
Isso não bloqueia o PR, mas é para você ficar atenta :smile: